.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,255 @@
+      options:
+        - label: This site DOES NOT contains sexually explicit material;
+          required: true
+        - label: Filters were updated before redroduced an issue;
+          required: true
+        - label: AdGuard product version is up-to-date;
+          required: true
+        - label: Browser version is up-to-date;
+          required: true
+        - label: If the site or app is broken, disabling AdGuard protection resolves an issue.
+          required: true
+  - type: dropdown
+    id: adguards
+    attributes:
+      label: What product do you use?
+      multiple: true
+      options:
+        - AdGuard Browser Extension
+        - AdGuard Content Blocker
+        - AdGuard DNS
+        - AdGuard for Android
+        - AdGuard for iOS
+        - AdGuard for Mac
+        - AdGuard for Safari
+        - AdGuard for Windows
+        - Other ad blocker
+    validations:
+      required: true
+
+  - type: input
+    id: version
+    attributes:
+      label: AdGuard version
+      description: What version of our software are you running?
+      placeholder: ex. 7.7
+    validations:
+      required: true
+
+  - type: dropdown
+    id: what-happened
+    attributes:
+      label: What type of problem have you encountered?
+      description: If the problem does not fall under any category that is listed here, please contact our tech support - support@adguard.com
+      multiple: true
+      options:
+        - Missed ads or ad leftovers
+        - Website or app doesn't work properly
+        - AdGuard gets detected on a website
+        - Social media buttons — share, like, tweet, etc.
+        - Annoyances — pop-ups, cookie warnings, etc.
+        - Missed analytics or tracker
+        - Dangerous site
+    validations:
+      required: true
+
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browsers is used?
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
+        - Opera
+        - Yandex Browser
+        - other(specify in comment below)
+        - the problem is with app, not a site
+    validations:
+      required: true
+
+  - type: dropdown
+    id: browsers_type
+    attributes:
+      label: Which device is used?
+      multiple: false
+      options:
+        - Desktop
+        - Mobile
+    validations:
+      required: true
+
+  - type: input
+    id: url
+    attributes:
+      label: Where did you encounter the problem?
+      description: Please enter address of the page or download link of the affected app.
+      placeholder: ex. https://example.com/
+    validations:
+      required: true
+
+  - type: dropdown
+    id: filters
+    attributes:
+      label: What filters do you have enabled?
+      description: Select enabled filters from the list or enter URLs of any custom filters in comment below.
+      multiple: true
+      options:
```

Comment:
let's divide full list of filters to AdGuard (required) and third-party (optional) and add optional text-area for custom filters

```
  - type: dropdown
    id: agfilters
    attributes:
      label: What AdGuard filters do you have enabled?
      description: Select enabled AdGuard filters from the list.
      multiple: true
      options:
        - AdGuard Base filter
        - AdGuard Mobile Ads filter
        ....
        - AdGuard Experimental filter
        - Filter unblocking search ads and self-promotion
    validations:
      required: true

  - type: dropdown
    id: thirdpartyfilters
    attributes:
      label: What third-party filters do you have enabled?
      description: Select enabled third-party filters from the list.
      multiple: true
      options:
        - EasyList
        - ABPindo
        - Bulgarian list
        - EasyList China
        - EasyList Czech and Slovak
        - EasyList Dutch
        - EasyList Germany
        - EasyList Hebrew
        - EasyList Italy
        - EasyList Lithuania
        - Latvian List
        - Liste AR
        - Liste FR
        - ROList
        - EasyPrivacy
        - Icelandic ABP List
        - AdBlockID
        - Greek AdBlock Filter
        - Fanboy's Annoyances
        - Fanboy's Social Blocking List
        - Web Annoyances Ultralist
        - Thai Ads Filters
        - Hungarian filter
        - Peter Lowe's Blocklist
        - Xfiles
        - Adblock Warning Removal List
        - Online Malicious URL Blocklist
        - Spam404
        - RU AdList - Counters
        - ABPVN List
        - Fanboy's Enhanced Tracking List
        - Official Polish filters for AdBlock, uBlock Origin & AdGuard
        - Polish GDPR-Cookies Filters
        - Estonian List
        - ChinaList+EasyList
        - CJX's Annoyances List
        - Polish Social Filters
        - Adblock-Persian list
        - Fanboy's Swedish
        - AdGuard Chinese filter
        - Fanboy's Anti-Facebook List
        - Fanboy's Vietnamese
        - List-KR
        - xinggsf
        - I don't care about cookies
        - Fanboy's Spanish\/Portuguese
        - EasyList Spanish
        - KAD - Anti-Scam
        - Adblock List for Finland
        - ROLIST2
        - Iranian filter
        - road-block
        - Polish Annoyances Filters
        - Polish Anti Adblock Filters
        - Fanboy's Anti-thirdparty Fonts
        - BarbBlock
        - EasyList Cookie List
        - NoCoin Filter List
        - Frellwit's Swedish Filter
        - YousList
        - AlleBlock
        - EasyList Polish
        - Polish Anti-Annoying Special Supplement
        - Dandelion Sprout's Nordic Filters
        - Dandelion Sprout's Annoyances List
        - Legitimate URL Shortener
    validations:
      required: false

  - type: textarea
    id: customfilters
    attributes:
      label: Custom filters
      description: |
        Enter custom filters URLs if you use them
      placeholder: |
        enter custom filters URLs
    validations:
      required: false
```

Reply:
too complicated. We have reporting tool.